addons/hw_drivers/websocket_client.py

```diff
@@ -13,6 +13,23 @@
 _logger = logging.getLogger(__name__)
 websocket.enableTrace(True, level=logging.getLevelName(_logger.getEffectiveLevel()))
 
+
+class WebsocketMessageHandlers:
```

Comment:
using a singleton class with only staticmethod is quite smart. But I do wonder if there is not something simpler like putting a bunch of functions in a dedicated file (a bit like the helper.py) and use getattr on it ?
See: https://stackoverflow.com/questions/53529034/using-classes-with-only-static-methods-for-organizational-purposes#:~:text=I%20would%20just%20use%20the%20module%20itself.%20Modules%20are%20essentially%20static%2Dclasses%20which%20are%20singleton%27s%20to%20boot.%20I%20suppose%20it%20comes%20down%20to%20a%20matter%20of%20taste%2C%20but%20a%20class%20with%20all%20staticmethods%20screams